src/main/java/com/bobcat00/viaversionstatus/Listeners.java

```diff
@@ -234,81 +234,116 @@ public void onPlayerJoin(PlayerJoinEvent e)
-                            plugin.config.getNotifyString().replace("%player%",      player.getName()).
-                            replace("%displayname%", player.getDisplayName()).
-                            replace("%version%",     clientVersion).
-                            replace("%server%",      serverVersion)));
+            if (!player.hasPermission("viaversionstatus.exempt.notify.message")) {
+                for (Player p : Bukkit.getServer().getOnlinePlayers()) {
+                    if (p.hasPermission("viaversionstatus.notify")) {
+                        p.sendMessage(ChatColor.translateAlternateColorCodes('&',
+                                plugin.config.getNotifyString().replace("%player%", player.getName()).
+                                                                replace("%displayname%", player.getDisplayName()).
+                                                                replace("%version%", clientVersion).
+                                                                replace("%server%", serverVersion)));
+                    }
                 }
             }
-            
-            String notifyCommand = plugin.config.getNotifyCommand();
-            if (!notifyCommand.isEmpty())
-            {
-                notifyCommand = notifyCommand.replace("%player%",      player.getName()).
-                        replace("%displayname%", player.getDisplayName()).
-                        replace("%version%",     clientVersion).
-                        replace("%server%",      serverVersion);
-                plugin.getLogger().info("Executing command " + notifyCommand);
-                try
-                {
-                    Bukkit.getServer().dispatchCommand(Bukkit.getConsoleSender(), notifyCommand);
-                }
-                catch (CommandException exc)
-                {
-                    plugin.getLogger().info("Command returned exception: " + exc.getMessage());
+
+            if (!player.hasPermission("viaversionstatus.exempt.notify.command")) {
+                String notifyCommand = plugin.config.getNotifyCommand();
+                if (!notifyCommand.isEmpty()) {
+                    notifyCommand = notifyCommand.replace("%player%", player.getName()).
+                                                  replace("%displayname%", player.getDisplayName()).
+                                                  replace("%version%", clientVersion).
+                                                  replace("%server%", serverVersion);
+                    plugin.getLogger().info("Executing command " + notifyCommand);
+                    try {
+                        Bukkit.getServer().dispatchCommand(Bukkit.getConsoleSender(), notifyCommand);
+                    } catch (CommandException exc) {
+                        plugin.getLogger().info("Command returned exception: " + exc.getMessage());
+                    }
```

Comment:
Please keep braces on separate lines as in the original.

Reply:
Oops, sorry about that. My text editor sort of did that on its own and I didn't notice it while looking over my commits before pull request submission ;;

Addressed in commits `a5c1f4d949e4eeeb4ffea94fb2c8d71cb17687d6` and also `7085b29a8f2cf3f195921601681383dac47aba65`.